src/analysis/retail/paladin/holy/modules/heroTalents/herald/Dawnlight.tsx

```diff
@@ -0,0 +1,210 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/paladin';
+import SpellIcon from 'interface/SpellIcon';
+import SpellLink from 'interface/SpellLink';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent, HealEvent } from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
+class Dawnlight extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  protected combatants!: Combatants;
+
+  healingSource = new Map<
+    number,
+    {
+      totalAmount: number;
+      totalOverheal: number;
+      hits: { timestamp: number; amount: number; overheal: number }[];
+    }
+  >();
```

Comment:
```suggestion

type HealingInfo {
  totalAmount: number;
  totalOverHealk;
  hits: {timestamp: number, amount: number, overheal:number}[]
}

class Dawnlight extends Analyzer {
  static dependencies = {
    combatants: Combatants,
  };
  protected combatants!: Combatants;

  healingSource = new Map<
    number,
    HealingInfo
  >();
```

define a type outside of the class and use that (even better if you plan to do this sort of thing in multiple files then put it into a common file.